src/main/java/org/opentripplanner/routing/algorithm/mapping/RaptorPathToItineraryMapper.java

```diff
@@ -264,6 +286,27 @@ private boolean isFree(EgressPathLeg<T> egressPathLeg) {
     return egressPathLeg.egress().isFree();
   }
 
+  /**
+   * If a routing result transfers at the very same stop RAPTOR doesn't add a path leg. However,
+   * sometimes we want to create a zero distance leg so the UI can show a transfer. Since it would
+   * be considered backwards-incompatible, this is an opt-in feature.
+   */
```

Comment:
```suggestion
  /**
   * If a routing result transfers at the very same stop, RAPTOR doesn't add a path leg. However,
   * sometimes we want to create a zero distance leg so a UI can show a transfer. Since it would
   * be considered backwards-incompatible, this is an opt-in feature.
   */
```